person.rb

```diff
@@ -0,0 +1,24 @@
+require './student'
```

Comment:
### Optional
- [ ] It would be better if we change the `require` to `require_relative` since we are referencing files in our currently directory

Reply:
Thank you so much I was not very sure on what to use. I appreciate